ci-operator/README.md

```diff
@@ -0,0 +1,154 @@
+ci-operator
+===========
+
```

Comment:
Pro-tip: always start any doc with a one-liner describing the doc's purpose & audience. e.g. "This describes how to configure end-to-end tests using CI Operator, and is intended for OpenShift component developers who want to get their code under test." Or alternatively: "After reading this doc, you will know: what CI Operator is, how to configure it, where to find out more information about it, ...."

I believe doing this greatly helps the reader (to know if the doc is worth investing time in) and the author (to improve the focus and understand how much they need to support their audience by explaining concepts).

Reply:
Added.